ATTACK/Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    private GameObject[] spawnPointsT1;
+    private GameObject[] spawnPointsT2;
+
+    public GameObject character;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        spawnPointsT1 = GameObject.FindGameObjectsWithTag("Team1Spawn");
+        spawnPointsT2 = GameObject.FindGameObjectsWithTag("Team2Spawn");
+
+        for (int i = 0; i < 3; i++)
+        {
+            SpawnCharacter(character, spawnPointsT1, i, 0);

```

Comment:
Please don’t use 0 and 1 magic numbers as team number. Create some enum/struct for readability

Reply:
Really good point